core/node/storage/deprecated_pg_stream_store.go

```diff
@@ -0,0 +1,1624 @@
+package storage
```

Comment:
Let's introduce config parameter to enable putting new streams into the new tables,
then there fill be if statement in CreateStream, and this entire file is not going to be needed.

The reason for config parameter is to control testing and rollout on deployed clusters: 
we don't want operators to transition immediately without testing it first. Without config feature flag we'll have to delay entire release if testing is not complete.